weatherAplication/Modules/CustomView/CustomAlert.swift

```diff
@@ -0,0 +1,12 @@
+import Foundation
+import UIKit
+
+extension UIViewController {
+    func showCustomAlert(description: String) {
+        DispatchQueue.main.async {
+            let alert = UIAlertController(title: R.string.localizable.error(), message: description, preferredStyle: .alert)
+            alert.addAction(UIAlertAction(title: R.string.localizable.ok(), style: .default))
+            self.present(alert, animated: true)
+        }
+    }
+}
```

Comment:
hmm lepiej to extension do pliku o innej nazwie wpakowac jesli robisz to w ten sposob, np UIViewControllerExtensions albo UIKitExtensions

Reply:
mozesz na tym samym poziomie co Modules i Model zrobic folder Helpers gdzie wrzucisz taki plik